queries.sql

```diff
@@ -68,3 +70,25 @@ WHERE date_of_birth
 BETWEEN '1990-01-01' 
 AND '2000-12-31' 
 GROUP BY species;
+
+-- Project 3
+--Join Query by owner name
+select * from animals JOIN owners on animals.owner_id=owners.id where owners.full_name='Melody Pond';
+
+--Join Query by species name
+select * from animals JOIN species on animals.species_id=species.id where species.name='Pokemon';
+
+--Full Join Query by owner
+select * from animals Full JOIN owners on animals.owner_id = owners.id;
+
+--Join Count group by species
+select species.name as animals_species,count(species.name) as species_count from species JOIN animals on animals.species_id=species.id GROUP BY species.name;
```

Comment:
- [ ] Please show the number of pokemon (4) and digimon (6), not a list of them. That's the requirement 👇🏽 

> How many animals are there per species?

Reply:
corrected

![Screenshot (55)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/01a39f48-4456-42ab-a98d-5421f50314ed)